### PR TITLE
update upi config

### PR DESCRIPTION
**Pull request by mjkkirschner**

### CODE REVIEW

**Review by reddyashish — APPROVED** [review]

**Comment by QilongTang** [comment]
I dont think release info needs to be updated, the UPI `release` is different than our understanding of `release`. You probably want to refer to previous hotfix definitions, like https://github.com/DynamoDS/Dynamo/blob/RC2.16.1_master/tools/autobuild/build.json

**Comment by mjkkirschner** [comment]
ok @QilongTang thanks for pointing that out - it seems I really only need to update the build.json file?

File: src/Config/upiconfig.xml [hunk 1]
```diff
@@ -8,17 +8,17 @@
         <upi_attribute name='id' value='DYN'/>
         <upi_element name='level'>
           <upi_attribute name='name' value='release'/>
-          <upi_attribute name='id' value='2.17.0'/>
+          <upi_attribute name='id' value='2.17.1'/>
```
Comment (pinzart90) [inline src/Config/upiconfig.xml:11]: release needs to remain `2.17.0`

Only the build id tag is supposed to be changed when incrementing the build part of the version

**Comment by mjkkirschner** [comment]
PTAL!

**Review by pinzart90 — APPROVED** [review]

**Pull request merged**